components/PeerReviews/PeerReviewScoreSummary.tsx

```diff
@@ -0,0 +1,92 @@
+import { ReactElement } from "react";
+import { PeerReviewScoreSummary } from "~/config/types/peerReview";
+import { StyleSheet, css } from "aphrodite";
+import { breakpoints } from "~/config/themes/screen";
+import ScoreInput from "~/components/Form/ScoreInput";
+import ALink from "~/components/ALink";
+import colors from "~/config/themes/colors";
+
+type Props = {
+  summary: PeerReviewScoreSummary,
+  docUrl: string,
+};
+
+function PeerReviewSummary({
+  summary,
+  docUrl = "",
+}: Props): ReactElement {
+  return (
+    <div className={css(styles.reviewContainer)}>
+      <span className={css(styles.reviewScoreContainer)}>
+        <span className={css(styles.reviewScore)}>{summary?.avg}</span>/10
+      </span>
+      <div className={css(styles.scoreContainer)}>
+        <span className={css(styles.dot)}>&bull;</span>
+        <ScoreInput
+          value={summary?.avg}
+          readOnly={true}
```

Comment:
This evaluates to `true`
```suggestion
          readOnly
```